gpu-operator/life-cycle-policy.rst

```diff
@@ -141,12 +143,20 @@ Refer to :ref:`Upgrading the NVIDIA GPU Operator` for more information.
    * - NVIDIA GDRCopy Driver
      - `v2.5.0 <https://github.com/NVIDIA/gdrcopy/releases>`__
 
-.. _gds-open-kernel:
+.. _known-issue:
 
    :sup:`1`
+   Known Issue: For drivers 570.124.06, 570.133.20, and 570.148.08, 
```

Comment:
@tariq1890, will all these driver versions work with 25.3.0? 

I'm just thinking that we don't list them as a supported driver, will it cause more confusion to have the complete list of impacted drivers, or to just say the ones we list as supported are impacted?

Reply:
we also dont list 570.86.15 as supported, should we?

...sorry if i'm over thinking this :)

Reply:
That's a very valid question, thanks for highlighting this! I think it would make sense to add those TRD versions back to the Platform support and then mention `570.86.15` as the driver version unaffected by this bug.

Reply:
I think it's ok. We should update the docs once the other driver has been released.